### PR TITLE
Remove Preset support

### LINKED ISSUE #249: Deprecate and then remove the Form Presets
### N+1 version

- [x] Deprecate the Form Presets usage,
  - [x] Raise a DeprecationWarning at loading time,
  - [x] Add a deprecation warning in the swagger documentation
  - [x] Add a deprecation warning in the documentation

### N+2 version

- [x] Remove the Presets models and logic
  - [x] ... what shall we do with the demo ``formidable.js`` code?
  - [x] Remove models and thus, all the business logic,
  - [x] Remove from translations,
  - [x] Remove from Swagger definition,
  - [x] ~~Django migration to remove tables~~ handled by #255

### PR DESCRIPTION
**Pull request by brunobord**
## Review

This PR closes #249, #95, #96, #172, #187

* [x] Tests
* [x] Docs/comments
  * [x] Docs
  * [x] run ``tox -e swagger-statics`` to rebuild the swagger static files **and** commit the diff.
* [x] Translations -- updated (a lot of strings are just gone)
* [x] Migration(s) -- added the DROP TABLE
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -72,7 +72,7 @@ crowdin-check:
 
 # target: crowdin-gettext-makemessages - generate .po files for all the available languages - NEEDS CROWDIN CREDENTIALS
 crowdin-gettext-makemessages: crowdin-check
-	cd formidable; ../.crowdin/bin/django-admin makemessages `.crowdin/bin/crowdin-cli-py list translations | cut -d '/' -f 1 | sed 's/.*/-l & /'`; cd ..
+	cd formidable; ../.crowdin/bin/django-admin makemessages --no-obsolete `.crowdin/bin/crowdin-cli-py list translations | cut -d '/' -f 1 | sed 's/.*/-l & /'`; cd ..
```
Comment (wo0dyn) [inline Makefile]: Why did you add this option?

File: formidable/forms/__init__.py [hunk 2]
```diff
@@ -133,28 +126,10 @@ def get_dynamic_form_class(formidable, role=None, field_factory=None):
 
     conditions_json = formidable.conditions or []
     attrs['_conditions'] = conditions_register.build(attrs, conditions_json)
-    attrs['rules'] = presets_register.build_rules(formidable, attrs)
     return type(str('DynamicForm'), (BaseDynamicForm,), attrs)
 
 
-class FormidableFormMetaclass(forms.Form.__class__):
```
Comment (wo0dyn) [inline formidable/forms/__init__.py]: 👋  Metaclass!

File: Makefile (hunk 1 above)
Comment (brunobord) [reply Makefile]: this option removes the obsolete strings, cleaning up the .po files.

Comment (wo0dyn) [reply Makefile]: OK. 👍
